lib/client.js

```diff
@@ -101,9 +101,10 @@ extend(Raven.prototype, {
     }
 
     this.onFatalError = this.defaultOnFatalError = function (err, sendErr, eventId) {
-      console.error(err.stack);
+      console.error(err && err.stack ? err.stack : err);
```

Comment:
out of curiosity, can you build this only with short circuiting and no ternary? :P